verifying-signatures.html

```diff
@@ -14,8 +14,33 @@
 </div>
 
 <div id="how-to-verify-signatures">
+    <h4>Download Options</h4>
+    <p>Every KeePassXC release is published in a variety of package formats:</p>
+    <ul>
+        <li>a <em>*.dmg</em> drag-and-drop installer for OS X</li>
+        <li>an <em>*.exe</em> installer and a <em>*.zip</em> archive with binaries for Windows</li>
+        <li>a self-contained executable <em>*.AppImage</em> for GNU/Linux.</li>
+        <li>a <em>*.tar.xz</em> source tarball</li>
+    </ul>
+    <p>You will need to download one of these to install on your machine.</p>
+
+    <h4>Sidecar files</h4>
+    <p>Each of these files has a related <em>*.sig</em> sidecar file that you
+       should download as well. These contain a GPG <em>signature</em> created with
+       one of our <em>release keys</em>. Signing the installable file with
+       any other key will give a different signature, so you can use the signature
+       and our public key to check the package file really came from us.</p>
+
+    <p>Each release package also has a related <em>*.DIGEST</em> file. Grab that too.
```

Comment:
I mentioned it here so the reader would be guided to get all the files they might need rather than having to go back and download it when they get to the 'digest' discussion. Also it seemed tidier to keep discussion of the downloadable files in one lump.

I take it your point is that the signature check also confirms the file was transferred without errors, making the digest check redundant? I suggested in the pr preamble comment that the whole business of the digest should perhaps be in a separate subsection before the verification stuff. So the sub-headings would be something like:
 * download options (mentions packages only)
 * basic integrity check (deals with .DIGEST and shasum)
 * checking both integrity and authenticity (describes .sig and gpg)
   * importing public key
   * verifying releases

Reply:
I'd be okay with that structure.

Reply:
Ok, I've pushed an extra couple of changes that should achieve this.